frontend/src/container/CreateAlertV2/EvaluationSettings/utils.tsx

```diff
@@ -0,0 +1,246 @@
+import { generateTimezoneData } from 'components/CustomTimePicker/timezoneUtils';
+import dayjs, { Dayjs } from 'dayjs';
+import timezone from 'dayjs/plugin/timezone';
+import utc from 'dayjs/plugin/utc';
+import { rrulestr } from 'rrule';
+
+import { WEEKDAY_MAP } from './constants';
+import { CumulativeWindowTimeframes, RollingWindowTimeframes } from './types';
+
+// Extend dayjs with timezone plugins
+dayjs.extend(utc);
+dayjs.extend(timezone);
```

Comment:
Do you need to add additional plugins. As I understand it seems like we need to add additional plugins for `week` handling?

Reply:
Could you please help me understand where exactly it might be needed? The EvaluationCadence component using it seems to working fine.

Reply:
Check docs once https://day.js.org/docs/en/plugin/weekday

Reply:
Verified. Not required in our case.